_episodes/04-tabular_data.md

```diff
@@ -307,31 +307,104 @@ Wait!  That's not right!  What is wrong with that output?  How can we fix it?
 {: .challenge}
 
 ## Plotting Data
-Another common way to analyze tabular data is to graph it.  To graph our data, we will need a new python library that contains functions to plot data.  
+Another common way to analyze tabular data is to graph it.  To graph our data, we will need a new python library that contains functions to plot data. To plot our data, we will use a Python library called <a href='https://matplotlib.org/index.html'>`matplotlib`</a>.
 
 ```
 import matplotlib.pyplot
 ```
 {: .language-python}
 ```
-fig_1 = matplotlib.pyplot.plot(data[:,0])
+plt.figure()
```

Comment:
whoops. you're right, `plt` hasn't been introduced at this point. It does initialize the figure. This isn't necessary if you are making only one figure, but using it allows you to make multiple plots in a `for` loop (later in the episode). Otherwise, it will keep adding to the same plot in the `for` loop.